client/web/compose/src/views/Admin/Pages/Builder.vue

```diff
@@ -425,6 +451,14 @@ export default {
         this.page.blocks.push(block)
       }
 
+      // if a new block has been added
+      if (this.page.blocks[this.page.blocks.length - 1].blockID === NoID) {
```

Comment:
Instead of checking it twice do it in the above ifs, since they check the exact same thing